Codeinwp/neve
# WooCommerece improvements

### Summary
1. Removed the word-break attribute so the table on checkout would look good
2. Increased size of Total on the cart page
3. Made the billing fields to be 100%

### Will affect visual aspect of the product
NO

### Screenshots 
<img width="762" alt="Screenshot at Feb 19 17-07-32" src="https://user-images.githubusercontent.com/9929553/108522046-0a386280-72d5-11eb-8d12-9dcedc90b152.png">
<img width="358" alt="Screenshot at Feb 19 17-08-41" src="https://user-images.githubusercontent.com/9929553/108522127-26d49a80-72d5-11eb-9eea-b152bbd003c8.png">
<img width="470" alt="Screenshot at Feb 24 16-43-21" src="https://user-images.githubusercontent.com/9929553/109017086-852bbf80-76bf-11eb-8850-ad55627c8c61.png">

### Test instructions
- Create a shop with some products with really long names. Make sure it's not a name with no space in it like "thisproducthasareallylongnameandnospaceinitbecausewewanttotest" because it won't look good, I removed the break-word property that caused the issue
- Check if the subtotal and the prices for products are still squashed.
  
  
- On the cart page, check if the total is bigger and is in bold.


Closes #2469, #2432, #2547.
